core/components/atoms/icon/icons.json

```diff
@@ -253,7 +253,9 @@
       "height": 20
     },
     "brand-yandex": {
-      "paths": ["M8.814 20v-6.488L4.5 1.875h2.169l3.18 8.973L13.505 0H15.5l-4.692 13.586V20z"],
+      "paths": [
```

Comment:
Changes on this file don't seem to be related to the dependency update. Can we remove them?

Reply:
Sorry about that. Fixed in e105b923. I think the icons.dev task may have updated that.